### ISSUE TITLE
Support umbrella projects

### ISSUE THREAD

**Issue by jeremyowensboggs** [OvermindDL1/gradualixir#1]
When run in an umbrella project, gradualixer gives the following error
```
** (RuntimeError) trying to access Mix.Project.app_path for an umbrella project but umbrellas have no app
    (mix) lib/mix/project.ex:495: Mix.Project.app_path/1
    (mix) lib/mix/project.ex:521: Mix.Project.compile_path/1
    lib/mix/tasks/gradualizer.ex:42: Mix.Tasks.Gradualizer.run_gradualizer/2
    lib/mix/tasks/gradualizer.ex:25: Mix.Tasks.Gradualizer.run/1
    (mix) lib/mix/task.ex:316: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:79: Mix.CLI.run_task/2
    (elixir) lib/code.ex:767: Code.require_file/2
```
Let me know, and I'll have a whack at it.
Thanks for this project, btw.

**Comment by OvermindDL1** [comment]
Ah, I entirely forgot about umbrella apps (I don't use them, I consider them a mis-design, so I forget they exist often...  ^.^;).

It should be an easy fix though.  Hold on and I'll do it shortly...  :-)

**Comment by OvermindDL1** [comment]
Fixed in ae461e8 along with added a feature to wildcard passed-in paths.  :-)

Please confirm the fix works.  :-)
